src/components/pages/AddPhrases.js

```diff
@@ -1,5 +1,22 @@
-import PhraseForm from "../PhraseForm.js";
+import PhraseForm from '../PhraseForm.js';
+import { useEffect } from 'react';
+import ModalPhraseAddedMessage from '../ModalPhraseAddedMessage.js';
 
-export default function AddPhrases({phrases, handlePhraseSubmit}) {
-  return <PhraseForm phrases={phrases} handlePhraseSubmit={handlePhraseSubmit} />
-}
\ No newline at end of file
+export default function AddPhrases({ phrases, handlePhraseSubmit, showMessage, setShowMessage }) {
+
+  useEffect(() => {
+    if (showMessage) {
+      setTimeout(() => {
+        setShowMessage(false);
+      }, 2500);
+    }
+  }, [setShowMessage, showMessage]);
+  return (
+    <>
+      {showMessage ? (
+        <ModalPhraseAddedMessage message="Dein Kind wurde erfolgreich hinzugefügt! Füge nun einen Spruch hinzu!" />
+      ) : null}
```

Comment:
```suggestion
      {showMessage && (
        <ModalPhraseAddedMessage message="Dein Kind wurde erfolgreich hinzugefügt! Füge nun einen Spruch hinzu!" />
      )}
```

Reply:
I was keeping track to this article...maybe I understood it wrong [Use ternaries rather than && in JSX](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)...that´s why I´m always placing ternary and with null !? why would u change it?

Reply:
You are not wrong :) just a different approach.
You will just need to make sure that the statement before the `&&` is `true` or `false` and then it won't render `0` or something like that. Here is another article about the topic: https://thoughtspile.github.io/2022/01/17/jsx-conditionals/
Maybe this will shed some light?

Reply:
ok cool thanks ;)